core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -759,17 +760,17 @@ private static String toSql(RelNode root, SqlDialect dialect,
     final String expected = "SELECT \"product_class_id\", COUNT(*) AS \"C\"\n"
         + "FROM \"foodmart\".\"product\"\n"
         + "GROUP BY ROLLUP(\"product_class_id\")\n"
-        + "ORDER BY \"product_class_id\", COUNT(*)";
+        + "ORDER BY \"product_class_id\", 2";
     final String expectedMysql = "SELECT `product_class_id`, COUNT(*) AS `C`\n"
         + "FROM `foodmart`.`product`\n"
```

Comment:
COUNT(*) is replaced by 2 rather than 1. So the ordinal starts from 1?

Reply:
Yes, I think the ordinal is the position of column in select list, it starts from 1.